Assets/Tests/InputSystem/CoreTests_Actions_Interactions.cs

```diff
@@ -231,6 +231,48 @@ public void Actions_WhenTransitioningFromOneInteractionToNext_GetCallbacks()
         }
     }
 
+    // https://jira.unity3d.com/browse/ISXB-310
+    [Test]
+    [Category("Actions")]
+    public void Actions_WhenTransitioningFromOneInteractionToNextAlreadyPerformed_GetCallbacksAndResetOtherInteractions()
+    {
+        ResetTime();
+
+        var gamepad = InputSystem.AddDevice<Gamepad>();
+
+        var action = new InputAction("test", InputActionType.Button, binding: "<Gamepad>/buttonSouth",
+            interactions: "multiTap(tapCount=3),multiTap,Tap");
```

Comment:
Is the first multitap interaction adding any value to this test - wondering if that unnecessary complexity should be removed?

Reply:
it's unfortunately necessary since the first one is special. We need a first one that doesn't complete are fallback to the second that already triggered.